presentation/common/src/presentation-common/hierarchy/Key.ts

```diff
@@ -274,3 +278,23 @@ export interface LabelGroupingNodeKeyJSON extends GroupingNodeKeyJSON {
  * @public
  */
 export type NodeKeyJSON = BaseNodeKeyJSON | ECInstancesNodeKeyJSON | ECClassGroupingNodeKeyJSON | ECPropertyGroupingNodeKeyJSON | LabelGroupingNodeKeyJSON;
+
+/**
+ * Data structure that describes an instance keys select query
+ * @beta
+ */
+export interface InstanceKeysSelectQuery {
+  /** The query to select instance keys */
+  query: string;
+  /** The query parameters */
+  bindings: string[];
```

Comment:
`bindings` aren't storing strings, are they?

Reply:
Changed to match native serializer side.